### ISSUE TITLE
unit time.midinote should rather be time.midipitch

### ISSUE THREAD

**Issue by vincentgoudard** [ossia/libossia#707]
IMHO, this would be more accurate WRT to what a "MIDI note" is, i.e. a tuple with pitch, velocity and channel information.

Apparently, this is an heritage from Jamoma, so question is, to @jln- @bltzr @theod @lossius @matcham @reno- and other people counting on Jamoma patches : would that break too much thing for you to consider the change ?

Benefit is only the clarity for new Score users like me I guess, but we could be many ;)

**Comment by bltzr** [comment]
I think we can have alternative names to units in libossia (can you confirm, @jcelerier ?), so we could change this the way you say (which makes perfect sense) and still have people using the old syntax (and not breaking their patches)

**Comment by jcelerier** [comment]
yep, adding it :) (would everyone be ok with also "pitch" as alias ?)

**Comment by vincentgoudard** [comment]
works for me — the simpler the better :)

**Comment by matcham** [comment]
doesn't break patches and makes Vincent happy, so I'm happy too :-)
